ground-control/internal/server/handlers.go

```
@@ -84,237 +53,696 @@ func (s *Server) healthHandler(w http.ResponseWriter, r *http.Request) {
+	if err != nil {
+		log.Println(err)
+		HandleAppError(w, err)
+		return
+	}
+	// Create a new Queries object bound to the transaction
+	q := s.dbQueries.WithTx(tx)
+	// Ensure proper transaction handling with defer
+	defer func() {
+		if p := recover(); p != nil {
+			// If there's a panic, rollback the transaction
+			tx.Rollback()
```

golangci-lint (1.62.2)

Error return value of `tx.Rollback` is not checked

(errcheck)